com.microsoft.mrtk.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -1,18 +1,19 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
 using Unity.Profiling;
 using UnityEngine;
 using UnityEngine.Events;
 using UnityEngine.InputSystem;
+using UnityEngine.XR.Interaction.Toolkit;
 using UnityPhysics = UnityEngine.Physics;
 
 namespace Microsoft.MixedReality.Toolkit.SpatialManipulation
 {
     /// <summary>
     /// Tap to place is a far interaction component used to place objects on a surface.
     /// </summary>
-    [RequireComponent(typeof(StatefulInteractable))]
     [AddComponentMenu("MRTK/Spatial Manipulation/Solvers/Tap To Place")]
     public class TapToPlace : Solver
     {
```

Comment:
placementActionReference is no longer used, should be removed (TapToPlaceEditor will need to be updated too)